apps/agreement/contracts/Agreement.sol

```diff
@@ -752,6 +753,17 @@ contract Agreement is IAgreement, AragonApp {
         return _isDisputed(challenge);
     }
 
+    /**
+    * @notice Check if a certain amount for a certain address can be unlocked
```

Comment:
Reading this out of context, this description sounds a bit vague and disconnected from the staking pool.